package.json

```diff
@@ -83,41 +79,20 @@
     "semver": "^7.3.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/preset-env": "^7.12.10",
-    "@babel/preset-react": "^7.12.10",
     "@netlify/eslint-config-node": "^6.0.0",
     "ava": "^4.0.0",
     "babel-loader": "^8.2.2",
     "c8": "^7.11.0",
     "cpy": "^9.0.0",
     "cpy-cli": "^4.0.0",
-    "cypress": "^7.0.0",
     "del": "^6.0.0",
-    "execa": "^6.0.0",
     "html-webpack-plugin": "^5.0.0",
     "husky": "^7.0.4",
-    "is-ci": "^3.0.0",
-    "node-static": "^0.7.11",
     "path-browserify": "^1.0.1",
-    "puppeteer": "^13.0.0",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
     "test-each": "^4.0.0",
     "tmp-promise": "^3.0.2",
-    "webpack": "^5.10.1",
-    "webpack-cli": "^4.2.0",
-    "webpack-dev-server": "^4.0.0"
+    "webpack-cli": "^4.10.0"
   },
-  "browserslist": [
-    "last 2 Chrome versions",
-    "last 2 Opera versions",
-    "last 2 Firefox versions",
-    "last 2 Edge versions",
-    "last 2 Safari versions",
-    "last 2 iOS versions",
-    "last 2 ChromeAndroid versions"
-  ],
```

Comment:
Given the demo app was removed from this project, it made sense to me that enforcement of this should likely from from the netlify FE project rather than here. If there's another reason why this should be added back in let me know